docs/_snippets/document-query-complete-tip.mdx

```diff
@@ -0,0 +1,6 @@
+<Tip>
+  There is a scenario in which a previous document query has already retrieved all of the available
+  documents for a patient, in which case, the `medical.document-download` webhook notification will
+  contain an empty document array: `documents: []`. Simply fetch all of the available documents with
+  [this endpoint](/medical-api/api-reference/document/get-document).
```

Comment:
this is verbose, and at this point the dev might now have a technical understanding of the webhooks - why not just add a quick line to the existing tip? ie

```
<Tip>
  To get updates about the document query progress, including download and conversion status, check
  out our [webhooks guide](/medical-api/more-info/webhooks).
  
  Note that the webhooks will only contain updates for new data fetched in the current document query.
</Tip>
```